.env.example

```
@@ -7,6 +7,9 @@ NEXT_PUBLIC_GITHUB_AUTH_TOKEN=__KEY__
 NEXT_PUBLIC_OLAS_MECH_SUBGRAPH_URL=__URL__
 NEXT_PUBLIC_OLAS_PREDICT_AGENTS_SUBGRAPH_URL=__URL__
 
+NEXT_PUBLIC_GNOSIS_ATA_SUBGRAPH_URL=__URL__
+NEXT_PUBLIC_BASE_ATA_SUBGRAPH_URL=__URL__
```

sorry @axatbhardwaj I struggle to define what are the subgraphs for these as I need to add the endpoints to Vercel. based on our discussions I guess they are the new mech marketplace? why don't we have the legacy one then here?
also can we rename them to 
```suggestion
NEXT_PUBLIC_GNOSIS_MM_SUBGRAPH_URL=__URL__
NEXT_PUBLIC_BASE_MM_SUBGRAPH_URL=__URL__
```

the same name we use in mm app so hopefully should be clear that MM is MechMarketplace